### PR TITLE
Fix variable @program typo in programs#show

### PR DESCRIPTION
**Pull request by differentreality**
Fixes failing test of features/program_spec.rb
I believe it was suceeding before, because the user was not really an organizer, but an admin.

### CODE REVIEW

**Comment by hennevogel** [comment]
You should rebase...

File: spec/factories/users.rb [hunk 1]
```diff
@@ -16,6 +16,10 @@
       gravida.
     EOS
 
+    after(:create) do |user|
```
Comment (hennevogel) [inline spec/factories/users.rb]: Why is this here?

Comment (differentreality) [inline spec/factories/users.rb]: I don't know, some bad git pull on master perhaps.

**Comment by sonalkr132** [comment]
hey, can we get rid bc1d16a from this PR?

**Comment by ChrisBr** [comment]
LGTM, merging... :+1:
